pom.xml

```diff
@@ -45,17 +44,119 @@
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
 
+<!--		<dependency>-->
```

Comment:
Remove commented out code. Same for the below dependencies as well.

Reply:
Done